core/conn/jdbc_type2/native/CSrvrConnect.cpp

```diff
@@ -457,6 +461,23 @@ SRVR_STMT_HDL *SRVR_CONNECT_HDL::getSrvrStmt(long dialogueId,long stmtId,long
     FUNCTION_RETURN_PTR(NULL,(NULL));
 }
 
+SRVR_STMT_HDL *SRVR_CONNECT_HDL::getInternalSrvrStmt(long dialogueId, const char* stmtLabel, long *sqlcode)
+{
+    FUNCTION_ENTRY("getInternalSrvrStmt",("dialogueId=0x%08x, stmtLabel=0x%08x, sqlcode=0x%08x",
+                dialogueId,
+                stmtLabel,
+                sqlcode));
+
+    SRVR_STMT_HDL *pSrvrStmt = NULL;
+
+    MapOfInternalSrvrStmt::iterator iterOfStmtId = mapOfInternalSrvrStmt.find(stmtLabel);
+    if( !(iterOfStmtId == mapOfInternalSrvrStmt.end()) )
+        pSrvrStmt =(SRVR_STMT_HDL *)iterOfStmtId->second;
+
+    if(pSrvrStmt != NULL)
+        FUNCTION_RETURN_PTR(pSrvrStmt,(NULL));
+}
```

Comment:
What happens if pSrvrStmt is NULL? (I don't see a return statement for this case.)